api/src/main/java/org/apache/iceberg/types/TypeUtil.java

```diff
@@ -42,6 +42,45 @@
   private TypeUtil() {
   }
 
+  /**
+   * Project extracts particular fields from a schema. Unlike
```

Comment:
We typically have a short high-level description (hopefully on a single line) and then a paragraph break followed by other relevant information.